### DISCUSSION TITLE
Adding new shortcut

### DISCUSSION THREAD

**Discussion by AlienChamp** [bromite/bromite#1121] [General]
Hi, firstly I would like say that the bromite browser is by far the best and clean browser ever, I love it.

My question is how I can add more shortcuts in my home page.
Thanks 
![Screenshot_20210430_100704_org bromite bromite](https://user-images.githubusercontent.com/83447395/116674018-04799000-a99c-11eb-9af3-a7b1705790f4.jpg)

**Comment by AlienChamp** [comment]
Sorted, thanks

**Reply by csagan5** [reply]
What was the solution?

**Reply by AlienChamp** [reply]
If you use the site 3/4 times will pop in automatically. I normally copy link, paste then close. X3 and that's it. 😊
